Police hats are now hats instead of helmets.

JohnFulpWillard opened the pull request:
## About The Pull Request

Makes police hats a subtype of Warden's hats, so they still are security-level armor, without covering your hair like helmets do.
This was gone unnoticed because the item is very limited in-game. I've been in this place before https://github.com/tgstation/tgstation/pull/66872 ...

I also renamed the gravity generator updatepaths because it's been inconsistent with the rest of the updatepaths for a while now.

## Why It's Good For The Game

I hate it when a hats that don't cover your head end up turning you bald.

## Changelog

:cl:
fix: Police hats no longer turn you bald while worn, and no longer act like helmets, since they're caps.
/:cl:

JohnFulpWillard committed:
Fixes police hats covering your hair.

Makes police hats a subtype of Warden's hats, so they still are security-level armor, without covering your hair like helmets do.
This was gone unnoticed because the item is very limited in-game.

I also renamed the gravity generator updatepaths because it's been inconsistent with the rest of the updatepaths for a while now.

JohnFulpWillard committed:
renames updatepaths

Kapu1178 commented:
this works but you can also just remove the HIDEHAIR flag

JohnFulpWillard commented:
> this works but you can also just remove the HIDEHAIR flag

I don't want to do that because I hate snowflake solutions.

san7890 commented on tools/UpdatePaths/Scripts/69643_police_hat.txt:
```diff
@@ -0,0 +1 @@
+/obj/item/clothing/head/helmet/police : /obj/item/clothing/head/warden/police
```
```suggestion
/obj/item/clothing/head/helmet/police : /obj/item/clothing/head/warden/police{@OLD}
```

Did you update the DMM manually?

JohnFulpWillard committed:
Update tools/UpdatePaths/Scripts/69643_police_hat.txt

Co-authored-by: san7890 <the@san7890.com>

ShizCalev commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -191,7 +191,7 @@
 	dog_fashion = null
 
-/obj/item/clothing/head/helmet/police
+/obj/item/clothing/head/warden/police
```
pssst.

helmet.dm

Iamgoofball commented:
sorry bro im just gonna say it, it's more snowflake to repath it to warden helmet than to make it hide the hair

ShizCalev commented:
![image](https://user-images.githubusercontent.com/6209658/188242533-7f9d0d1c-4e24-45d5-82f4-0eb01e7d46dd.png)
![image](https://user-images.githubusercontent.com/6209658/188242553-5ffb5e84-aad2-4c19-a236-49daa59ae796.png)
![image](https://user-images.githubusercontent.com/6209658/188242592-03522ef4-f13e-4a30-b7d0-6fd98110a120.png)
balance change?

JohnFulpWillard committed:
moves the hat to warden's, removes icon state

Turns out the icon state is shared between both hats, so the only real difference is name and description...

JohnFulpWillard commented:
> sorry bro im just gonna say it, it's more snowflake to repath it to warden helmet than to make it hide the hair

The sprite is a hat, it's called a hat, I don't see why it should be treated like a helmet.
I also found out they share the same icon and icon state as the warden's default hat, so the only difference between the two was their name, and description.
